lighthouse/environment/src/lib.rs

```diff
@@ -229,6 +231,24 @@ impl<E: EthSpec> Environment<E> {
 
     /// Sets the logger (and all child loggers) to log to a file.
     pub fn log_to_json_file(&mut self, path: PathBuf) -> Result<(), String> {
+        // Creating a backup if the logfile already exists.
+        if path.exists() {
+            let start = SystemTime::now();
+            let timestamp = start
+                .duration_since(UNIX_EPOCH)
+                .expect("Time went backwards")
```

Comment:
I think we should return an `Err` here instead of a panic. Unlikely to see it, but it's good hygiene IMO.

Reply:
Yup good point!